release-1.6/release-notes-draft.md

```diff
@@ -58,12 +58,33 @@
 
 ## Cluster Lifecycle
 ### New Features
+- kubeadm is a simple tool for creating Kubernetes clusters on existing
+  machines. kubeadm's command line interface is now in beta. This means that
+  using it is considered safe.
+  Please [try it out](https://kubernetes.io/docs/getting-started-guides/kubeadm/) and
+  [give us feedback](https://kubernetes.io/docs/getting-started-guides/kubeadm/#feedback)!
+
 ### Notable Changes
-- New Bootstrap Token authentication and management method. Works well with kubeadm.
-- Adds a new cloud-controller-manager binary that may be used for testing the new out-of-core cloudprovider flow
-- Introduces an API for the Kubelet to request TLS certificates from the API server.
-- kubeadm is enhanced and improved with a baseline feature set and command line flags that are now marked as beta.
+- kubeadm 1.6 sets up a more secure cluster by default.  It uses [RBAC](https://kubernetes.io/)
```

Comment:
How about  "_now_ sets up" a more secure cluster by default.  Otherwise it isn't clear what you are comparing it to.